src/Literals/Boolean.purs

```diff
@@ -0,0 +1,13 @@
+module Literals.Boolean where
```

Comment:
Let's add export list. Also, add `toBoolean`

Reply:
I'm not sure about `toBoolean`. Should I also introduce `BooleanLiteral` here like:
```purescript
foreign import kind Boolean
foreign import data True ∷ Boolean
foreign import data False ∷ Boolean

foreign import BooleanLiteral :: Boolean -> Type
```
plus something like:

```purescrpit
toBoolean :: forall a. BooleanLiteral a -> Boolean
```

Have I missed the point? ;-)